test/Config.hs

```diff
@@ -0,0 +1,14 @@
+module Config (testConfig) where
```

Comment:
Let's rename this module to `Test.Config` and move it under `test/Test/Config.hs` path to introduce some sort of namespacing